driver/mounter.go

```diff
@@ -214,47 +230,41 @@ func (m *mounter) Unmount(target string) error {
 	return mount.CleanupMountPoint(target, m.kMounter, true)
 }
 
-func (m *mounter) IsRunning(source string, cmdContext execContext) bool {
-	cmd := fmt.Sprintf("ls -l %s | awk '{print $NF}'", source)
-	out, err := cmdContext("/bin/sh", "-c", cmd, source).CombinedOutput()
-
+func (m *mounter) IsRunning(av iAttachmentValidator, source string) bool {
```

Comment:
Can we make `iAttachmentValidator` a member/field of `mounter`? This would hide the dependency injection more elegantly and keep the `IsRunning` signature simpler.